contributors/devel/sig-architecture/alt-arch.md

```diff
@@ -0,0 +1,64 @@
+# Shipping alternate architectures in Kubernetes release artifacts
```

Comment:
nit: Am I the only one to be bothered by not having `CPU instruction sets` written here? I have the impression that the "architecture" word might lead to a different meaning for different people. So being clear in the title would help, but it's maybe me...

Reply:
Yeah its an internal term AFAIK. It expresses both Operating system and CPU instructionset. as illumos/amd64 and linux/amd64 are also reffered as architectures, even though illumos is a different OS on the same CPU ISA.

Reply:
Thanks for clarifying! Is that understood by the average audience? :)

Reply:
We've been using the term "architecture" the same way on multiple projects that are being upstreamed for RISC-V and ARM. It's broadly understood to my knowledge.

Reply:
Kubernetes is a go project, in go terminology arch/architecture refers to e.g. amd64
https://golang.org/pkg/runtime/#pkg-constants
https://golang.org/pkg/go/build/#Context